src/array.jl

```diff
@@ -193,20 +193,11 @@ CategoricalVector(::UndefInitializer, m::Integer;
                   levels::Union{AbstractVector, Nothing}=nothing,
                   ordered::Bool=false) =
     CategoricalArray(undef, m, levels=levels, ordered=ordered)
-CategoricalVector{T}(::UndefInitializer, m::Int;
```

Comment:
Why aren't these needed anymore? Sounds like good news!

Reply:
With the new definition of `CategoricalVector` and `CategoricalMatrix` these definitions are redundant; they correspond to methods above with R specified.